Source/SwiftLintFramework/Rules/LineLengthRule.swift

```diff
@@ -87,4 +91,23 @@ public struct LineLengthRule: ConfigurationProviderRule, SourceKitFreeRule {
         return modifiedString
     }
 
+    private func stripUrls(fromSourceString sourceString: String) -> String {
```

Comment:
also, for Swift 3 API guidelines, this should probably be `stripURLs(from string: String) -> String` or a `fileprivate` extension on `String` as a computed variable `strippingURLs`.